docs_src/microservices/device/details/AutoEvents.md

```diff
@@ -5,23 +5,68 @@ title: Device Service - Auto Events
 # Device Service - Auto Events
 ## Concept
 ![image](Auto_Events.png)
```

Comment:
In the new diagram, the second Core Metadata on the right should be App Services.

Reply:
oh yeah absolutely, thanks! updated